package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "make clean",
     "help": "make help",
     "refresh": "make refresh",
-    "test": "make test"
+    "test": "mocha -R list"
```

Comment:
@rmg TY, I didn't notice the `-R` was for reporter, thought it was `-r` for recursive. Checked the docs anyways and yes @bajtos is correct that it should not specify a reporter.